### PR TITLE
Add pagination to OH requests; remove "three month summary" filter.

### PR DESCRIPTION
**Pull request by eddierubeiz**
Ref #2511.

### CODE REVIEW

**Comment by jrochkind** [comment]
When we first implemented, we asked if they wanted pagination, and the staff at the time said no, they would only use this page to look at recent stuff, and if they wanted old stuff they'd use the export. 

What about the new filters seem to you to work best with pagination? I'm not following, sorry!

**Comment by jrochkind** [comment]
I think the use case for the new filters is to be able to focus on the 'pending' ones, and make sure they haven't missed any. 

Ah, if there WAS a "pending" one from earlier than last 3 months, i guess they wouldn't want to miss it!

Is that what you were thinking?

**Comment by eddierubeiz** [comment]
Yeah, that's basically what i was thinking. I might be mistaken, but doing exactly what the issue asks for might lead to some confusion. Let's not worry about it until we merge the other PR - if I am wrong we can ignore this PR.

**Comment by jrochkind** [comment]
Yeah, you're right, it makes sense to me too. just to have to clear it with OH staff to make sure it does to them too. Show it on staging to them? I dunno, I leave to you?

File: app/controllers/admin/oral_history_access_requests_controller.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@ def index
     unless status == "any" || status.blank?
       scope = scope.where(delivery_status: status)
     end
-    @oral_history_access_requests = scope.where('created_at > ?', 3.months.ago).order(created_at: :desc).to_a
+    @oral_history_access_requests = scope.order(created_at: :desc).page(params[:page]).per(100)
```
Comment (jrochkind) [inline app/controllers/admin/oral_history_access_requests_controller.rb:12]: We were showing way more than 100 before I think, should we make it 200 or 300 any thoughts?

**Comment by jrochkind** [comment]
omg i love how little code is required. thanks kaminari

**Pull request merged**